Build failure on NetBSD

Build failed on NetBSD 8.0 as below.

~~~
CC    ../../polarssl/library/net_sockets.c -> build/host/mrbgems/mruby-polarssl/polarssl/library/net_sockets.o
/home/vagrant/mruby-polarssl/polarssl/library/net_sockets.c: In function 'mbedtls_net_accept':
/home/vagrant/mruby-polarssl/polarssl/library/net_sockets.c:319:29: error: storage size of 'client_addr' isn't known
     struct sockaddr_storage client_addr;
~~~

--- a/mrbgem.rake
+++ b/mrbgem.rake
@@ -9,6 +9,7 @@ MRuby::Gem::Specification.new('mruby-polarssl') do |spec|
   spec.cc.include_paths << "#{polarssl_src}/../../mruby-io/include"
   spec.cc.include_paths << "#{build.root}/src"
   spec.cc.flags << '-D_FILE_OFFSET_BITS=64 -Wall -W -Wdeclaration-after-statement'
+  spec.cc.flags << '-D_NETBSD_SOURCE' if RUBY_PLATFORM =~ /netbsd/i
 
   spec.objs += Dir.glob("#{polarssl_src}/library/*.{c,cpp,m,asm,S}").map { |f| f.relative_path_from(dir).pathmap("#{build_dir}/%X.o") }
 

